README.md

```diff
@@ -168,7 +168,7 @@ You can specify sliced inference parameters as:
 sahi predict --slice_width 256 --slice_height 256 --overlap_height_ratio 0.1 --overlap_width_ratio 0.1 --model_confidence_threshold 0.25 --source image/file/or/folder --model_path path/to/model --model_config_path path/to/config
 ```
 
-- Specify postprocess type as `--postprocess_type UNIONMERGE` or `--postprocess_type NMS` to be applied over sliced predictions
+- Specify postprocess type as `--postprocess_type GREEDYNMM` or `--postprocess_type NMS` to be applied over sliced predictions
```

Comment:
Maybe we call it `GMS` ? In the paper they used the term Greedy Merge Strategy, we can state a reference for GMS as well. Wdyt ? @fcakyon

Reply:
@devrimcavusoglu which paper do you mention about? can you give url?

Reply:
[OverFeat](https://arxiv.org/abs/1312.6229) paper, afaik they introduced this merging strategy first in their paper (as a paper though earlier works couldve been done possibly). I don't know if later the name changed to something more common.

Reply:
in that paper, they take the average of bbox coordinates when merging, we are taking the union of the predictions. 

Moreover, they merge only bounding boxes while we also merge the predicted segmentation masks.

I think we can give a different name since its not the same technique.

Reply:
Oh I mistaken our method as merging by averaging, just like the paper. NMM sounds good now :smile: :+1: